transforms/this-render-migration/index.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const { getParser } = require('codemod-cli').jscodeshift;
+const { addImportStatement, writeImportStatements } = require('../utils');
+
+module.exports = function transformer(file, api) {
+  const j = getParser(api);
+  const root = j(file.source);
+
+  /**
+   * Replace deprecated this.render() with render() from '@ember/test-helpers' package
+   */
+  function transform() {
+    root.find(j.ExpressionStatement, {
+      expression: {
+        type: 'AwaitExpression',
```

Comment:
I think we don't have to look for an `AwaitExpression` here. I vaguely remember that I already have used `render()` without immediately awaiting for it (just some rare edge case for sure).

So instead I think we can just look for a `CallExpression` matching `this.render`, and replace it with a `CallExpression` of `render`, whether it has an `AwaitExpression` as its parent node or not. So basically remove the wrapping `AwaitExpression` in both the `j.find()` call and the builder call below.

Reply:
Make sense! Updated the pull request.